content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -40,7 +40,7 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
 
 - **Discussions and questions**: #dev-experience channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
 - **[Support](#support)**: `@dev-experience-support` in Slack
-- **GitHub**: [team/dev-experience](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdev-experience) label and @sourcegraph/dev-experience team.
+- **GitHub**: [team/dev-experience](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdev-experience) label and [@sourcegraph/dev-experience team](https://github.com/orgs/sourcegraph/teams/dev-experience).
```

Comment:
GitHub team mentions are also generated, so link not needed here either :) https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#contact

![image](https://user-images.githubusercontent.com/23356519/149383760-6307967b-fb7f-463b-b9bf-2e9362eca7a4.png)

Reply:
I love this company.